File: translations/codenet/uncommented/C2C++/gpt-4o/Code/s736909850.cpp
#include <iostream>

int main() {
    int sheep;
    int wolf;

    std::cin >> sheep >> wolf;

    if (wolf >= sheep) {
        std::cout << "unsafe" << std::endl;
    } else {
        std::cout << "safe" << std::endl;
    }

    return 0;
}

// <END-OF-CODE>

File: translations/codenet/uncommented/C2J/granite/Code/s736909850.java

import java.util.Scanner;

public class s736909850{
   public static void main(String[] args) {
      Scanner scanner = new Scanner(System.in);
      int sheep = scanner.nextInt();
      int wolf = scanner.nextInt();

      if (wolf >= sheep) {
         System.out.println("unsafe");
      } else {
         System.out.println("safe");
      }
   }
}
// END-OF-CODE